client/question/components/CodeSnippetDirective.js

```diff
@@ -65,7 +65,7 @@ tie.directive('codeSnippet', [function() {
          */
         $scope.openCodeModal = function() {
           MonospaceDisplayModalService.showModal(
-            'Code History', $scope.snippetLines);
+            'Previous Submission', $scope.snippetLines);
```

Comment:
I was discussing this with eyurko awhile back and he mentioned the likelihood that users (students in particular) will hesitate when they see "submit". How about "Previous Code" or something like that?

Reply:
Also consistent with us moving away from the "Submit" button.

Reply:
"Previous Code" sounds fine to me. I'll change that. Thanks!